psydac/api/tests/test_api_2d_fields.py

```diff
@@ -58,22 +58,13 @@ def run_poisson_2d(filename, solution, f):
     F  = element_of(V, name='F')
 
     # Bilinear form a: V x V --> R
-    a = BilinearForm((u, v), integral(domain, dot(grad(u), grad(v))))
+    a = BilinearForm((u, v), integral(domain, u*v))
 
     # Linear form l: V --> R
-    l = LinearForm(v, integral(domain, f * v))
-
-    # Dirichlet boundary conditions
-    bc = [EssentialBC(u,  0, B_dirichlet_0)]
-
+    l = LinearForm(v, integral(domain, solution * v))
 
     # Variational model
-    equation = find(u, forall=v, lhs=a(u, v), rhs=l(v), bc=bc)
-
-    # Error norms
-    error  = u - solution
-    l2norm = Norm(error, domain, kind='l2')
-    h1norm = Norm(error, domain, kind='h1')
+    equation = find(u, forall=v, lhs=a(u, v), rhs=l(v))
```

Comment:
Please add a comment:
```python
# uh is the L2-projection of the analytical field "solution"
```
I would also change the name of of the variable "solution" with something else

Reply:
done